src/consumption.cpp

```diff
@@ -14,10 +14,13 @@
 #include "mutation.h"
 #include "options.h"
 #include "translations.h"
+#include "units.h"
 
 #include <string>
 #include <algorithm>
 
+using namespace units::literals;
```

Comment:
Again, basing our actions on the standard Library is not necessarily the best course. The units header is small enough that its inclusion should imply use of the literals.
Anyway that's not holding up this PR.